### PR TITLE
Added New Website to learn programming languages free online

### PR DESCRIPTION
**Pull request by himanshuchandola**
Added Code Learning Free Courses Online Platform

### CODE REVIEW

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for contributing @himanshuchandola ! Please add the new topic in the content list in the beginning as well

**Review by himanshuchandola — COMMENTED** [review]
Added under Others.

**Comment by himanshuchandola** [comment]
Topic is now listed under Others in the content list

File: README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ To add a resource or contribute, please check out our contribution guide [here](
   - [Design](#design)
   - [Open Source](#open-source)
   - [Standard Template Library](#standard-template-library)
+  - [Programming Courses Free Online](#learn-to-code)
```
Comment (shahednasser) [inline README.md:61]: Please make the titles the same, thank you!

**Comment by himanshuchandola** [comment]
Done

File: README.md [hunk 2]
```diff
@@ -533,3 +534,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 ### Standard Template Library
 
 - [C++ Standard Template Library in Practice](https://www.udemy.com/course/c-standard-template-library-in-practice/learn/)
+
+### Programming Courses Free Online
+
+- [Programming Courses Free Online](https://www.sololearn.com/learning)
```
Comment (shahednasser) [inline README.md:540]: This should be the title of the resource you're providing, so in this case the title should be "SoloLearn"

File: README.md [hunk 3]
```diff
@@ -58,7 +58,7 @@ To add a resource or contribute, please check out our contribution guide [here](
   - [Design](#design)
   - [Open Source](#open-source)
   - [Standard Template Library](#standard-template-library)
-  - [Programming Courses Free Online](#programming-courses-free-online)
+  - [SoloLearn](#programming-courses-free-online)
```
Comment (shahednasser) [inline README.md:61]: This one should be the title of the section (Programming Courses Free Online). What's under that section should be the title of the resource (in this case, SoloLearn).

Comment (himanshuchandola) [reply README.md]: I have Changed Title of content to "Programming Courses Free Online" and section title the same.
I think it is fine now

**Comment by shahednasser** [comment]
Great thank you for your work!

**Pull request merged**